server/src/services/album.service.ts

```diff
@@ -200,7 +201,7 @@ export class AlbumService extends BaseService {
 
     const removedIds = results.filter(({ success }) => success).map(({ id }) => id);
     if (removedIds.length > 0) {
-      await this.albumRepository.update({ id, updatedAt: new Date() });
+      await this.albumRepository.update(id, { id, updatedAt: new Date() });
```

Comment:
the updatedAt is now set in the repository, so I don't think you need it here anymore.